roles/openstack-source/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: "python prerequities for {{ project_name }}"
+  pip: name={{ item.name }} version={{ item.version }}
+  with_items: python_requirements
+  when: python_requirements is defined and item.version is defined
+
+- name: "python prerequities for {{ project_name }}"
+  pip: name={{ item.name }}
+  with_items: python_requirements
+  when: python_requirements is defined and item.version is not defined
```

Comment:
You should drop a comment in here about collapsing them down when the default(omit) feature is available :)